assets/test/specs/form.spec.js

```diff
@@ -24,6 +29,9 @@ var setup = function () {
   $textInputExample = $('#text-input-example');
   $radioYesElement = $('#radio-yes');
   $radioNoElement = $('#radio-no');
+  $radioGroupThis = $('#radio-this');
+  $radioGroupThat = $('#radio-that');
```

Comment:
I think the issue is that `this` and `that` are ambiguous and you need to refer to the fixture to see what they are. Can you rename them to be more descriptive?